lib/appsignal/config.ex

```diff
@@ -86,6 +89,32 @@ defmodule Appsignal.Config do
     map
   end
 
+  defp skip_session_data_backwards_compatibility(config, nil) do
+    Map.put_new(config, :send_session_data, true)
+  end
+
+  defp skip_session_data_backwards_compatibility(config, skip_session_data) do
+    IO.warn(
+      "appsignal: Deprecation warning: The `skip_session_data` config option is " <>
+        "deprecated. Please, use `send_session_data` instead."
+    )
+
+    case Map.has_key?(config, :send_session_data) do
+      true ->
```

Comment:
```suggestion
    if Map.has_key?(config, :send_session_data) do
```

Use an if-statement here.